test/jdk/java/awt/Frame/GetGraphicsStressTest/GetGraphicsStressTest.java

```diff
@@ -27,7 +27,7 @@
 
 /**
  * @test
- * @bug 8235638 8235739
+ * @bug 8235638 8235739 8285094
```

Comment:
It is not necessary to add bugid here, since this list is useful to verify the old bugs if the test will be reworked later.